articles/azure/asz-sco

```diff
@@ -0,0 +1,262 @@
+
+- Azure Table Storage
+Table storage is Microsoft's NoSQL key/attribute store - it has a schemaless design, making it different from traditional relational databases. With a schemaless data store, it's easy to adapt your data as the needs of your application evolve. Table storage is easy to use, so developers can create applications quickly.
+Table storage is a key-attribute store, which means that every value in a table is stored with a typed property name. The property name can be used for filtering and specifying selection criteria. A collection of properties and their values comprise an entity. Since table storage is schemaless, two entities in the same table can contain different collections of properties, and those properties can be of different types.
+You can use table storage to store flexible datasets, such as user data for web applications, address books, device information, and any other type of metadata that your service requires. For today's Internet-based applications, NoSQL databases like table storage offer a popular alternative to traditional relational databases.
+A storage account can contain any number of tables, and a table can contain any number of entities, up to the capacity limit of the storage account.
+
+> [!NOTE]
+> All storage in Azure Stack is taken from a common pool. When designing your service for UKCloud for Microsoft Azure, consider the best usage of your storage. For example, if you have a large object storage requirement, consider using the UKCloud's Cloud storage platform as part of your solution. 
+ 
+## Service Management
+- API 
+
+- UKCloud for Microsoft Azure Marketplace – provides access to pre-created images which help speed up application delivery 
+
+- Azure Resource Manager – allows customers to write templates to deploy, update and delete UKCloud for Microsoft Azure resources in a single coordinated operation 
+
+- VM Extensions – UKCloud for Microsoft Azure supports virtual machine extensions that provide post deployment management for automation (for example, Docker) 
+ 
+## Network
+
+UKCloud manage the physical firewalls that face public and secure networks.
+
+Customers can deploy mutliple gateway devices and configure the following using either the Azure Stack Portal GUI or API:
+
+- Virtual networks – UKCloud for Microsoft Azure allows customers to configure virtual private clouds, and allows them to create and manage their private address space within Azure. This can be configured via the UI, API and other tooling 
+- Load balancers (IPv4 only) – customers can deploy basic load balancers o VPN gateway – the UKCloud for Microsoft Azure VPN gateway can be used to create site-to-site VPNs to remote infrastructure 
```

Comment:
you can deploy

There's a random o (load balancers o VPN gateway)

you can use the UKCloud for Microsoft Azure VPN gateway to create site-to-site VPNs to remote infrastructure